configs/mspc.json

```diff
@@ -41,8 +49,5 @@
       "type"
     ]
   },
-  "conversation_id": [
-    "1181088484"
-  ],
```

Comment:
Could you please restore the `conversation_id`?

Reply:
Sure, though I am not sure what is the `conversation_id` and how it is set, is the current value correct? (i.e., `1181088484`)

Reply:
Hey, yes it is the correct value. We use it internally to match a config to its owner when they apply for DocSearch.

Reply:
Thanks! I reverted that change.